test/fixtures/time_records.yml

```diff
@@ -0,0 +1,15 @@
+# Read about fixtures at http://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
+
+one:
```

Comment:
I couldn't find where are this fixtures are used. 
If you describing two object with the same attributes, you should use some kind of template instead.

Reply:
pretty sure it was created by some kind of generators, Rspec or bin/rails maybe